src/lib/updating-element.ts

```diff
@@ -529,22 +546,32 @@ export abstract class UpdatingElement extends HTMLElement {
   /**
    * Updates the element. By default this method reflects property values to attributes.
    * It should be overridden to render and keep updated DOM in the element's root.
-   * Within `update()` setting properties does not trigger `invalidate()`, allowing
-   * property values to be computed and validated before DOM is rendered and updated.
-   * This means in an override of `update()`, before calling `super.update()`
-   * setting properties will not trigger another update, but after calling `super.update()`
-   * setting properties will trigger another update.
+   * Setting properties inside this method will *not* trigger another update.
    * * @param _changedProperties Map of changed properties with old values
    */
-  protected update(_changedProperties: PropertyValues): void {
+  protected update(_changedProperties: PropertyValues) {
     if (this._reflectingProperties !== undefined && this._reflectingProperties.size > 0) {
       for (const [k, v] of this._reflectingProperties) {
         this._propertyToAttribute(k, this[k as keyof this], v);
       }
       this._reflectingProperties = undefined;
     }
-    // Before this (before calling super)...
-    this._markUpdated();
+  }
+
+  /**
+   Invoked whenever the element is updated. Implement to perform
+   post updating tasks via DOM APIs, for example, focusing an element.
+   Setting properties inside this method will trigger the element to update.
```

Comment:
Add a newline before the last sentence, here and below. And maybe add some more detail after "to update", like "to update against this update cycle is complete"

Reply:
Added.